src/js/gi/components/profile/Outcomes.jsx

```diff
@@ -36,6 +36,15 @@ class Outcomes extends React.Component {
         </div>
 
         <div className="medium-12 column">
+          <svg className="graph-key" viewBox="0 0 650 50">
```

Comment:
Sure. I think I'll keep the component within this module though, since it's only used here.